cluster-up/ssh.sh

```diff
@@ -40,7 +40,7 @@ if [[ $provider_prefix =~ okd.* ]]; then
         exit 1
     fi
     shift
-    ssh -lcore -p $port core@127.0.0.1 -i ${ssh_key} $@
+    ssh -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no -q -lcore -p $port core@127.0.0.1 -i ${ssh_key} $@
```

Comment:
I wasn't sure if we wanted to do that here or not, I added those in the caller of this script. The same thing should happen in the non OKD version